### PR TITLE
#71 - enrich violations with account and region

### PR DESCRIPTION
**Pull request by mrandi**

### CODE REVIEW

File: fullstop-plugin-api/src/main/java/org/zalando/stups/fullstop/violation/ViolationStoreDomain.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+ */
+
+package org.zalando.stups.fullstop.violation;
+
+import com.google.common.base.MoreObjects;
+
+/**
+ * @author mrandi
+ */
+public class ViolationStoreDomain {
```
Comment (jbellmann) [inline fullstop-plugin-api/src/main/java/org/zalando/stups/fullstop/violation/ViolationStoreDomain.java]: Why not only 'Violation'?

Comment (mrandi) [inline fullstop-plugin-api/src/main/java/org/zalando/stups/fullstop/violation/ViolationStoreDomain.java]: no reason

File: fullstop-plugins/fullstop-keypair-plugin/src/main/java/org/zalando/stups/fullstop/plugin/keypair/KeyPairPlugin.java [hunk 2]
```diff
@@ -69,7 +71,8 @@ public void processEvent(final CloudTrailEvent event) {
 
         List<String> keyNames = CloudtrailEventSupport.containsKeyNames(event.getEventData().getRequestParameters());
         if (!CollectionUtils.isEmpty(keyNames)) {
-            violationStore.save(String.format("KeyPair must be blank, but was %s", keyNames));
+            violationStore.save(new ViolationStoreDomain(event.getEventData().getAccountId(),
+                    event.getEventData().getAwsRegion(), format("KeyPair must be blank, but was %s", keyNames)));
```
Comment (jbellmann) [inline fullstop-plugins/fullstop-keypair-plugin/src/main/java/org/zalando/stups/fullstop/plugin/keypair/KeyPairPlugin.java]: Maybe you should use CloudTrailEventSupport and its static methods to just write 'getRegion(event)'.

File: fullstop-plugin-api/src/main/java/org/zalando/stups/fullstop/violation/ViolationStoreDomain.java (hunk 1 above)
Comment (jbellmann) [inline fullstop-plugin-api/src/main/java/org/zalando/stups/fullstop/violation/ViolationStoreDomain.java]: So you will change it to 'Violation'? It's simpler to talk about 'Violation's with non-technical colleagues.

Comment (mrandi) [inline fullstop-plugin-api/src/main/java/org/zalando/stups/fullstop/violation/ViolationStoreDomain.java]: :+1:

File: fullstop-plugins/fullstop-keypair-plugin/src/main/java/org/zalando/stups/fullstop/plugin/keypair/KeyPairPlugin.java (hunk 2 above)
Comment (mrandi) [inline fullstop-plugins/fullstop-keypair-plugin/src/main/java/org/zalando/stups/fullstop/plugin/keypair/KeyPairPlugin.java]: :ok_hand:

**Pull request merged**